### PR TITLE
Update alchemy trainers gossip

### PR DESCRIPTION
**Pull request by NickTyrer**
This PR syncs vmangos gossips with cmangos gossips for alchemy trainers:
 - updates condition 209 from cmangos to 25 to sync with vmangos
 - adds valentines gossip to the new gossip_menu in vmangos

### CODE REVIEW

**Comment by NickTyrer** [comment]
@ratkosrb I have aligned this PR with the latest changes. I have given my reasoning for some of the changes in the sql comments (although I am probably wrong).

File: sql/migrations/20200516095255_world.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4201, 5316, 0),
+(4201, 5317, 361),
+(4201, 5314, 362);
+
+
+-- update gossip menu for tel'athir (I believe this is correct as he should not share the same gossip_menu_id with cyndra kindwhisper)
+UPDATE `creature_template` SET `gossip_menu_id` = 4111 WHERE `entry` = 5500;
+
+-- add correct gossip_menu_entries for tel'athir
```
Comment (ratkosrb) [inline sql/migrations/20200516095255_world.sql]: Tel'Athir uses gossip menu id 4112, not 4111.

File: sql/migrations/20200516095255_world.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+-- add correct gossip_menu_entries for lilyssia nightbreeze (I believe this is correct as one of the broadcasts mentions the npc next to her)
+DELETE FROM `gossip_menu` WHERE `text_id` = 5016 AND `entry` = 4201;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4201, 5316, 0),
+(4201, 5317, 361),
+(4201, 5314, 362);
+
+
+-- update gossip menu for tel'athir (I believe this is correct as he should not share the same gossip_menu_id with cyndra kindwhisper)
+UPDATE `creature_template` SET `gossip_menu_id` = 4111 WHERE `entry` = 5500;
```
Comment (ratkosrb) [inline sql/migrations/20200516095255_world.sql]: He has 4112 in sniff. But I don't have a sniff of Cyndra Kindwhisper, so she is probably not supposed to use 4112.

File: sql/migrations/20200516095255_world.sql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+-- add correct gossip_menu_entries for alchemist narett
+UPDATE `gossip_menu` SET `condition_id` = 362 WHERE `entry` = 4122 AND `text_id` = 5050;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4122, 5051, 25),
+(4122, 5053, 361);
+
+
+-- add correct gossip_menu_entries for lilyssia nightbreeze (I believe this is correct as one of the broadcasts mentions the npc next to her)
+DELETE FROM `gossip_menu` WHERE `text_id` = 5016 AND `entry` = 4201;
```
Comment (ratkosrb) [inline sql/migrations/20200516095255_world.sql]: Her gossip has 4201-5016 pair in sniff.

File: sql/migrations/20200516095255_world.sql [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `condition_id`) VALUES
+(4201, 5316, 0),
```
Comment (ratkosrb) [inline sql/migrations/20200516095255_world.sql]: This text seem to be for when player has too low skill to train with her. Should have condition.

```
My teachings would be too advanced for you at this stage. If you wish to learn more of potions, speak with Tel'Athir here instead.
```

**Pull request merged**